tests/test_examples.py

```diff
@@ -62,6 +63,27 @@ def test_SOLOVEV_results(SOLOVEV):
     np.testing.assert_allclose(theta_err, 0, atol=1e-4)
 
 
+@pytest.mark.regression
+@pytest.mark.solve
+def test_SOLOVEV_anisotropic_results(SOLOVEV):
```

Comment:
This is good, but it doesn't actually test that solving with non-zero anisotropy works correctly. Do we have any benchmark cases we could use for that?

Reply:
Not really. Frank and I are working on comparing to near axis stuff but still in progress.